app/config.rb

```diff
@@ -20,12 +21,18 @@
 AppConfig[:indexer_log] = "/archivesspace/logs/archivesspace.out"
 AppConfig[:indexer_log_level] = "warn"
 
+## Plug-ins to load. They will load in the order specified
+#AppConfig[:plugins] = ['local', 'lcnaf'] (default)
+
+AppConfig[:allow_password_reset] = false
 AppConfig[:allow_user_registration] = false
+AppConfig[:authentication_restricted_by_source] = true
 AppConfig[:plugins] << "aspace-oauth"
 oidc_issuer = ENV["OIDC_ISSUER"]
 oidc_client_id = ENV["OIDC_CLIENT_ID"]
 oidc_client_secret = ENV["OIDC_CLIENT_SECRET"]
-if oidc_issuer && oidc_client_id && oidc_client_secret
+oidc_end_session_endpoint = ENV["OIDC_END_SESSION_ENDPOINT"]
+if oidc_issuer && oidc_client_id && oidc_client_secret && oidc_end_session_endpoint
   puts "OIDC settings were found; adding them to the configuration"
   AppConfig[:authentication_sources] = [
     {
```

Comment:
Looks like we may need to add `slo_link: true` here.

Reply:
Going to fix it in the local plugin view.